src/js/components/profile-form/spouse.html

```diff
@@ -74,7 +74,7 @@
 							class="btn form-control"
 							ng-class="{'btn-success': ngModel.$valid, 'btn-danger': !ngModel.$valid}"
 							ng-click="addSpouse()">
-						<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Spouse
+						<span class="glyphicon glyphicon-plus" aria-hidden="true"></span> <span translate>Spouse</span>
```

Comment:
Yes, the space looks significant for layout, but not translation. How best to do that?